in_text/text_01_spatial.yml

```diff
@@ -0,0 +1,57 @@
+title: >-
+  Predicting water temperature in the Delaware River Basin: 1 River reach information for 456 river reaches
+
+abstract: >-
+  This dataset provides shapefile outlines of the 2,332 lakes that had temperature modeled as part of this study.
+  The format is a shapefile for all lakes combined (.shp, .shx, .dbf, and .prj files). A csv file of lake metadata is also included.
+  This dataset is part of a larger data release of lake temperature model inputs and outputs for 2,332 lakes in the U.S.
+  (https://doi.org/10.5066/P9I00WFR).
+
+cross-cites:
+
```

Comment:
I think you want to put that formal citation to the geofabric data release here, and then refer to it below

Reply:
Like this:

```yaml
  -
    authors: ['Roland Viger']
    title: >-
      GIS Features of the Geospatial Fabric for National Hydrologic Modeling
    pubdate: 2014
    link: http://dx.doi.org/doi:10.5066/F7542KMD
```

Reply:
oops, yes, meant to include a spot for the geospatial fabric. Thanks.